torchvision/prototype/__init__.py

```diff
@@ -2,4 +2,3 @@
 from . import features
 from . import transforms
 from . import utils
-from . import models
```

Comment:
What's the reason for removing the models from all the `__init__.py` files? This means you won't be able to grab the model builders using some of the utility methods on the tests.

Reply:
the commit that remove this actually just revert, I add it before when testing the utilities to get all modules for instance. But you are right, I think its better to have this import on __init__.py, wil add it back.